### DISCUSSION TITLE
How can I loop ascending and descending only two states when clicking sortable column?

### DISCUSSION THREAD

**Discussion by sthtnr** [TanStack/table#2478] [General] [answered]
Hi guys.

I'm using `useSortBy` hook, and I want to make sure that every time I click on a sortable column, it takes only two states, ascending and descending, but it seems there need to be un-sorted state between two states as far as looking docs and demo.

Is there a way to achieve the results I want?
Thanks in advance.

**Comment by jzbuchalski** [comment]
We would also find only a two state sorting useful, anyone know how to achieve this?

**Comment by tadger** [comment, marked as answer]
I believe you're looking for the `disableSortRemove` option (passed to `useTable`) documented [here](https://github.com/tannerlinsley/react-table/blob/master/docs/api/useSortBy.md), eg:
```javascript
const {
  // ...
} = useTable({
  columns,
  data,
  disableSortRemove: true
}, useSortBy);
```

**Reply by sthtnr** [reply]
Thanks for your comment!
This is exactly what I was looking for:)

**Comment by jzbuchalski** [comment]
@tadger , thanks for the quick response, missed it in the docs.  Works as advertised 👍
